kolibri/plugins/audio_mp3_render/assets/src/views/index.vue

```diff
@@ -155,7 +155,9 @@
         this.$refs.audio.currentTime = sum;
       },
 
-      updateTime() {
+      updateTime(e) {
+        // guard against the function being called after component is unmounted
+        if (!this.$refs.audio) return;
```

Comment:
actually, the changes are slightly different - so if you can delete this change that will be helpful. thanks!